src/users/users.ts

```diff
@@ -149,6 +154,24 @@ export class Users {
     return deserializeAuthenticationResponse(data);
   }
 
+  async authenticateUserWithTotp(
```

Comment:
```suggestion
  async authenticateWithTotp(
```

Some of the changes in the stack of PRs was to remove `User` from the methods since it's already scoped to the `users` scope of the SDK, shouldn't this method follow the same principle?

Reply:
It makes sense. Let me handle it in a new PR, because I need to change the other ones.